Src/AutoFixture/DataAnnotations/MinLengthAttributeRelay.cs

```diff
@@ -0,0 +1,51 @@
+﻿using System;
+using System.ComponentModel.DataAnnotations;
+using System.Linq;
+using System.Reflection;
+using Ploeh.AutoFixture.Kernel;
+
+namespace Ploeh.AutoFixture.DataAnnotations
+{
+    /// <summary>
+    /// Relays a request for a constrained string to a <see cref="ConstrainedStringRequest"/>.
+    /// </summary>
+    public class MinLengthAttributeRelay : ISpecimenBuilder
```

Comment:
Please register this type also in [`NoDataAnnotationsCustomization.cs`](https://github.com/AutoFixture/AutoFixture/blob/e366edd604a89895f6989cd9491b9f99a7cec60d/Src/AutoFixture/NoDataAnnotationsCustomization.cs) for consistency.